hardware/SysfsGPIO.h

```diff
@@ -6,24 +6,35 @@
 #include "DomoticzHardware.h"
 #include "../main/RFXtrx.h"
 
+struct gpio_info
```

Comment:
I don't know how many items the vector will hold, but instead of 'int' could you use uint8_t (0-255), specially for boolean values like direction,active,edge, id's